From 34c49c5c335e3c204d8714654180dc99b7e3381d Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Tue, 1 May 2018 18:02:43 +0200
Subject: [PATCH 2/2] Allow for 1 minute intervals

Some (legacy) configuration files used 1m interval. Add here as well.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
Upstream-Status: Pending
---
 src/activitymanager/activity/schedule/IntervalSchedule.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/activitymanager/activity/schedule/IntervalSchedule.cpp b/src/activitymanager/activity/schedule/IntervalSchedule.cpp
index 6a4f832..8124515 100644
--- a/src/activitymanager/activity/schedule/IntervalSchedule.cpp
+++ b/src/activitymanager/activity/schedule/IntervalSchedule.cpp
@@ -181,7 +181,7 @@ unsigned IntervalSchedule::stringToInterval(const char *intervalStr, bool smart)
     unsigned totalSecs = 0;
     unsigned total = 0;
 
-    static const unsigned allowed[] = { 5, 10, 15, 20, 30, 60, 180, 360, 720 };
+    static const unsigned allowed[] = { 1, 5, 10, 15, 20, 30, 60, 180, 360, 720 };
     static const int num_allowed = (sizeof(allowed) / sizeof(int));
 
     int i;
-- 
2.37.3.windows.1

